src/lib/OTA/OTA.h

```diff
@@ -119,6 +126,13 @@ typedef struct {
                     packageIndex: 6;
             uint8_t payload[ELRS8_MSP_BYTES_PER_CALL];
         } msp_ul;
+        /** PACKET_TYPE_MAV **/
+        struct {
+            uint8_t packetType: 2,
+                    packageIndex: 5,
```

Comment:
Just a note for the future.  We will need to be careful if we decide to increase the SS buffer size.  I think a max packageIndex of 32 limits the buffer to 160B for the STD OTA mode.